river/metrics/cluster/rmsstd.py

```diff
@@ -0,0 +1,137 @@
+import math
+
+from river import utils
+
+from . import base
+
+__all__ = ["MSSTD", "RMSSTD"]
+
+
+class MSSTD(base.InternalClusMetric):
+    """Mean squared standard deviation.
```

Comment:
Please provide a brief description of the metric.

Besides, I would capitalize the letters to better match the acronym.

Reply:
This is pretty similar to the definition of `MSSTD`, which only removes the `root` part! I have added a brief description on that!
Capitalizing the letters resolved!